### ISSUE TITLE
User Statistics crashes with ONLY_FULL_GROUP_BY

### ISSUE THREAD

**Issue by solomonty** [innotop/innotop#140]
innotop Ver 1.11.1, MySQL 5.7, ONLY_FULL_GROUP_BY

> localhost INDEX_TABLE_STATISTICS: DBD::mysql::st execute failed: Expression #3 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'information_schema.L.ROWS_READ' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by at /usr/bin/innotop line 7532.

innotop Ver 1.10.0, MySQL 5.6, ONLY_FULL_GROUP_BY

> localhost INDEX_TABLE_STATISTICS: DBD::mysql::st execute failed: 'information_schema.L.ROWS_READ' isn't in GROUP BY at /usr/bin/innotop line 7497.

ONLY_FULL_GROUP_BY  is now the default on MySQL 5.7. Here is an excerpt from [Server SQL Modes](http://dev.mysql.com/doc/refman/5.7/en/sql-mode.html)

> The default SQL mode in MySQL 5.7 includes these modes: ONLY_FULL_GROUP_BY, STRICT_TRANS_TABLES, NO_ZERO_IN_DATE, NO_ZERO_DATE, ERROR_FOR_DIVISION_BY_ZERO, NO_AUTO_CREATE_USER, and NO_ENGINE_SUBSTITUTION.

**Comment by yoku0825** [comment]
This causes here.
https://github.com/innotop/innotop/blob/master/innotop#L4502-L4505

**Comment by yoku0825** [comment]
I fixed it and sent Pull-Request.

```
$ ./innotop -S /tmp/mysql.sock
RO] User Statistics (? for help)                         localhost, 1h8m, 36.92 QPS, 2/0/0 con/run/cac thds, 5.6.29-76.2-log

DB         Table         Reads     Reads Via Idx  Changed  Chg X Idx
mysqlslap  t1            12844680                    5659       5659
mysql      user                12                       0          0
mysql      proxies_priv         4                       0          0
mysql      db                   4                       0          0
```

**Comment by yoku0825** [comment]
This issue should be closed because my patch had been merged into 1.11.4
